parquet-common/src/main/java/org/apache/parquet/bytes/MultiBufferInputStream.java

```diff
@@ -89,6 +91,15 @@ public long skip(long n) {
     return bytesSkipped;
   }
 
+  @Override
+  public void skipFully(long n) throws IOException {
+    if (current == null || n > length) {
+      throw new EOFException();
```

Comment:
ditto: some info in exception, better retain the original message: `Not enough bytes to skip: " + skipped + " < " + n)`

Reply:
Got it